span-normalizer/span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizerTest.java

```diff
@@ -160,6 +162,10 @@ public void testServiceNameAddedToEvent() throws Exception {
                 KeyValue.newBuilder()
                     .setKey(JaegerSpanNormalizer.OLD_JAEGER_SERVICENAME_KEY)
                     .setVStr("testService"))
+            .addTags(
```

Comment:
why is this needed?

Reply:
We added this to increase number of items in the taglist for testing